doc/plugins.dox

```diff
@@ -170,14 +171,22 @@ Geany's website at http://www.geany.org/Support/BuildingOnWin32.
 
 @section helloworld "Hello World"
 
-When writing a plugin, you will find a couple of functions or macros which are mandatory
-and some which are free to use for implementing some useful feature once your plugin
-becomes more powerful like including a configuration or help dialog.
+@note This section describes the new entry points for plugins introduced with Geany 1.26. A short
+summary of the legacy entry points is given at page @ but it should not be used any more.
+
+When writing a plugin, you will find a couple of functions which are mandatory and some which are
+free to use for implementing some useful feature once your plugin becomes more powerful like
+including a configuration or help dialog.
+
+@subsection beginning Beginnings of any Plugin
```

Comment:
"Beginnings" is not the right word, either "Starting" if its the first thing in the plugin or "Outline" or "Overview" if its the skeleton of the plugin.